pluto/template/atom.xml.erb

```diff
@@ -1,67 +1,52 @@
 <?xml version="1.0" encoding="utf-8" standalone="yes" ?>
 <feed xmlns="http://www.w3.org/2005/Atom">
 
-	<title><%= site.title %></title>
-	<link rel="self" href="https://fedoraproject.org/people/rss20.xml"/>
-	<link href="https://fedoraproject.org/people/"/>
-	<id>https://fedoraproject.org/people/rss20.xml</id>
-	<updated><TMPL_VAR date_iso></updated>
-	<generator uri="http://fedoraplanet.org/"><TMPL_VAR generator ESCAPE="HTML"></generator>
+    <title><%= CGI.escapeHTML(site.title) %></title>
+    <link rel="self" href="<%= CGI.escapeHTML(site.url) %>"/>
+    <link href="<%= CGI.escapeHTML(site.url) %>"/>
+    <id><%= CGI.escapeHTML(site.url) %>/rss20.xml</id>
+    <updated><%= CGI.escapeHTML(site.updated.to_datetime.rfc3339) %></updated>
+    <generator uri="https://feedreader.github.io/"><%= CGI.escapeHTML(Pluto.generator) %></generator>
 
-<%
+    <%
     items = site.items.latest.limit(50)
     ItemCursor.new( items ).each do |item, new_date, new_feed|
-%>
-	<entry<TMPL_IF channel_language> xml:lang="<TMPL_VAR channel_language>"</TMPL_IF>>
-		<title type="html"<TMPL_IF title_language> xml:lang="<TMPL_VAR title_language>"</TMPL_IF>><TMPL_VAR title ESCAPE="HTML"></title>
-		<link href="<TMPL_VAR link ESCAPE="HTML">"/>
-		<id><TMPL_VAR id ESCAPE="HTML"></id>
-		<updated><TMPL_VAR date_iso></updated>
-		<content type="html"<TMPL_IF content_language> xml:lang="<TMPL_VAR content_language>"</TMPL_IF>>
-                <TMPL_IF channel_face>
-			&lt;img src=&quot;<TMPL_VAR channel_face ESCAPE="HTML">&quot; <TMPL_IF channel_facewidth>width=&quot;<TMPL_VAR channel_facewidth>&quot; </TMPL_IF><TMPL_IF channel_faceheight>height=&quot;<TMPL_VAR channel_faceheight>&quot; </TMPL_IF> alt=&quot;&quot; style=&quot;float: right;&quot;&gt;
```

Comment:
There was width and height being set here. I've removed those for now, since we don't have variables to use for them. I'm not sure if removing is the best approach though: maybe some standard size should be set?